scripts/create-feature-annotations.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+import json
+import os
+import sys
+
+feature_map = json.load(open(sys.argv[1]))
+for metric, features in feature_map.items():
+    features = set([feature.replace("Feature:", "") for feature in features])
+    try:
+        # the telemetry feature is not what we're tracking
+        features.remove("Telemetry")
+    except KeyError:
+        pass
+    if len(features):
+        metric_path = os.path.join("annotations", "fenix", "metrics", metric)
```

Comment:
Note that this script is fenix-specific, but the name of the script doesn't make that clear.